app/CodeFormatter/PrettierCodeFormatter.php

```diff
@@ -25,14 +22,15 @@ class PrettierCodeFormatter extends CodeFormatter
     ];
 
     /**
-     * Config is taken automatically from <code>.prettierrc</code> file
+     * Config is taken automatically from <code>.prettierrc</code> file.
+     *
      * @param string $file
-     * {@internal <code>--loglevel silent</code> does not override exit codes}
+     *                     {@internal <code>--loglevel silent</code> does not override exit codes}
```

Comment:
php-cs-fixer does not like attributes below a @param and formats them as a description for the parameter 😆
We can fix this in a later step.

Reply:
Ah, the `@internal` annotation isn't important anyways. The comment does not add any value. Not overriding an exit code is expected behaviour.

PS: I wonder how often that param organisation and internal get mentioned 😄